src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Controllers/BankIdController.cs

```diff
@@ -51,6 +51,7 @@ public ActionResult Login(string returnUrl, string loginOptions)
 
         private BankIdLoginViewModel GetLoginViewModel(string returnUrl, string loginOptions, BankIdLoginOptions unprotectedLoginOptions, AntiforgeryTokenSet antiforgeryTokens)
         {
+            var initialStatusMessage = unprotectedLoginOptions.AutoLaunch ? MessageShortName.RFA13 : MessageShortName.RFA1;
```

Comment:
If we want to give the user more instructions that they need to open their app AND "scan the QR code", do we want to do that as part of RFA1 or should we create a new message RFA1QR?

Reply:
It should be a separate message I would say. Will see how I can implement that.

Reply:
Implemented now! :)